internal/testrunner/runners/asset/tester.go

```diff
@@ -146,7 +146,7 @@ func (r *tester) run(ctx context.Context) ([]testrunner.TestResult, error) {
 	for _, e := range expectedAssets {
 		rc := testrunner.NewResultComposer(testrunner.TestResult{
 			Name:       fmt.Sprintf("%s %s is loaded", e.Type, e.ID),
-			Package:    installedPackage.Name,
```

Comment:
`installedPackage.Name` is the name set in the `manifest.yml` , but this could be different from the package folder name. As it happens, for instance, with the `sql_input` package in the integrations repository ([link](https://github.com/elastic/integrations/blob/17379ba1593b2f1f133ca8892c97a9f11ccd4087/packages/sql_input/manifest.yml#L2)):
- Folder name: `sql_input`
- Package name in the manifest: `sql`